### PR TITLE
DragonFish updates

### PR DESCRIPTION
**Pull request by bitpushr**
**Description**

⚒️ Fixes  # 

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning
- [ ] I made sure the title starts with `feat(chart-name):`, `fix(chart-name):` or `chore(chart-name):`

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

File: src/content/docs/platforms/SCALE/migrations/dragonfish-storage.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Cobia to DragonFish Storage Migration
+---
+
+:::caution[DragonFish is not yet supported]
```
Comment (Ornias1993) [inline src/content/docs/platforms/SCALE/migrations/dragonfish-storage.md:5]:
```suggestion
:::caution[Execute all steps at once]
```

File: src/content/docs/platforms/SCALE/migrations/dragonfish-storage.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+This is a pre-release guide, TrueNAS SCALE 24.04 DragonFish is explicitly **NOT** yet supported by TrueCharts
```
Comment (Ornias1993) [inline src/content/docs/platforms/SCALE/migrations/dragonfish-storage.md:7]:
```suggestion
Its imperative to execute **ALL** steps in this guide in one sitting.
If you reboot or take any kind of time-off, we cannot guarantee your data and apps stay working
```

File: src/content/docs/platforms/SCALE/migrations/dragonfish-storage.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Continue with the OpenEBS section of our TrueNAS SCALE Getting Started [guide](/platforms/scale/guides/getting-started#openebs-setup) and once complete, continue as below.
+
+After installing the `OpenEBS` app from the above guide, run the following and you're done:
+
+`sudo curl -s https://truecharts.org/openebsmv.sh | bash`
+
+### Migrating to new storage
+
+We highly advise moving to the new storage backend for existing Apps as well. _At a later date we will update this with migration instructions as well._
+Until we have migration steps ready, existing apps will continue to work like normal.
```
Comment (Ornias1993) [inline src/content/docs/platforms/SCALE/migrations/dragonfish-storage.md:56]:
```suggestion
Sadly enough, we require re-installation of all Apps to be migrated to the new storage solution.

However, thanks to the great collaboration with HeavyBullets, you can use his TT-Migration tool, to automate most of the installation process:
https://github.com/Heavybullets8/TT-Migration
```

File: src/content/docs/platforms/SCALE/migrations/dragonfish-storage.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+---
+title: Cobia to DragonFish Storage Migration
+---
+
+:::caution[Execute all steps at once]
+
+Its imperative to execute **ALL** steps in this guide in one sitting.
+If you reboot or take any kind of time-off, we cannot guarantee your data and apps stay working
+
```
Comment (Ornias1993) [inline src/content/docs/platforms/SCALE/migrations/dragonfish-storage.md:9]:
```suggestion

It's also important to note that migration to Dragonfish **requires** reinstallation of all Apps. For which a automated solution is available in the guide as well.
Please read this migration guide carefully, *before* upgrading to dragonfish.
```

**Comment by bitpushr** [comment]
Oops. Forgot to 'git pull'.

**Pull request merged**